src/Forms/SudoModePasswordField.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace SilverStripe\Forms;
+
+use SilverStripe\Forms\FormField;
+
+class SudoModePasswordField extends FormField
+{
+    public const FIELD_NAME = 'SudoModePasswordField';
+    
+    protected $schemaComponent = 'SudoModePasswordField';
+
+    protected $extraClasses = [
+        'SudoModePasswordField',
+    ];
+
+    public function __construct($name, $title = null, $value = '')
```

Comment:
Why accept any args if we're just gonna ignore them?

Reply:
I prefer to not change method signatures of subclasses

Reply:
Updated, will remove all the args